tests/test_basic.py

```diff
@@ -344,3 +344,106 @@ def test_rbd_pvc(rook_cluster):
     rook_cluster.kubernetes.kubectl('delete pod csirbd-demo-pod')
     rook_cluster.kubernetes.kubectl('delete pvc rbd-pvc')
     rook_cluster.kubernetes.kubectl('delete sc rook-ceph-block')
+
+
+def set_storage_pattern(rook_cluster, pattern):
+    # create a working-copy of cluster.yaml
+    cluster_working = os.path.join(rook_cluster.ceph_dir, 'cluster_test.yaml')
+    cluster_original = os.path.join(rook_cluster.ceph_dir, 'cluster.yaml')
+
+    cluster_orig_f = open(cluster_original, 'r')
```

Comment:
(nit) Could use a `with` statement to avoid doing the closes manually.

Reply:
good point. I will change that.